src/sb-util.ts

```diff
@@ -1,3 +1,49 @@
-export function sb() {
-  console.log('sb-util')
-}
\ No newline at end of file
+import { Queryable, ScratchProjectOptions } from './abstracts';
+import { SpOptionsEmptyError, SpMultipleSourceOptionsError } from './errors';
+import { ProjectSource, AssetFetcher } from '../common';
+
+export class ScratchProject implements Queryable {
+	constructor(projectJSON: Object, assetFetcher?: any) {
+
+	}
+
+	query(selector?: String) {
+		return [];
```

Comment:
Should we go ahead and return an non-array iterable?

```js
return {[Symbol.iterator]() {return {next() {return {done: true}}}}};
```

or with an empty generator

```js
return {*[Symbol.iterator]() {}}
```